opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/ais/AccountListingService.java

```diff
@@ -28,7 +28,8 @@
 
     @Override
     protected void doValidate(DelegateExecution execution, Xs2aContext context) {
-        validator.validate(execution, extractor.forValidation(context));
+        context.setClassName(this.getClass().getName());
```

Comment:
It is better to pass Class (not name) directly to validator as an argument

Reply:
ok